anta/tests/routing/bgp.py

```diff
@@ -1404,3 +1404,141 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"The following BGP peers are not configured or have non-zero update error counters:\n{failures}")
+
+
+class VerifyBGPPeerPrefixes(AntaTest):
+    """Verifies BGP IPv4 peer(s) consistency of prefix(s) received and accepted in a BGP session.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the `prefixReceived` equals `prefixAccepted`, indicating that all received prefix(s) were accepted.
+    * Failure: The test will fail if the `prefixReceived` is not equal to `prefixAccepted`, indicating that some prefix(s) were rejected/filtered out or
+    peer(s) are not configured.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.routing:
+      bgp:
+        - VerifyBGPPeerPrefixes:
+            address_families:
+              - afi: ipv4
```

Comment:
I believe we have to check for specific peer for specific AFI/SAFI. Please  recheck.

Reply:
Updated. Thanks!